docs/calimero_sdk/3_bridge.mdx

```diff
@@ -0,0 +1,270 @@
+const calimeroPrivateKey = "ed25519:myaccount-private-key-on-calimero-side"; // note that this private key is different than the one in the example above since one Account does not share key pair across chains
+const keyPair = KeyPair.fromString(calimeroPrivateKey);
+
+const res = await bridge.ftWithdraw(
+    ChainType.CALIMERO, // the side where we are burning and withdrawing FTs, in this example we are burning tokens on the Calimero side and unlocking them on the Near side (inverse action to the one in the previous example).
+    accountId, // the account that is withdrawing FTs to the other side
+    keyPair, // key pair of the account that is withdrawing FTs
+    "dev-1668507284663-45605813374523.ft_deployer.q.calimero.testnet", // FT contract with Calimero bridged suffix
+    3 // the amount that we are withdrawing, in this example we are withdrawing the exact amount we bridged in the earlier example
+);
+```
+
+### Bridge NFTs to another chain
+
+``` nftBridge ``` transfers non-fungible tokens from the chain where a token is originally created to another chain. In this example, we will show how to transfer NFTs from the Near side to the Calimero side.
+
+```js
+const myAccountId = "myaccount.testnet";
+const myAccountPrivateKey = "ed25519:myaccount-private-key"; // here we should use the actual private key for the account that is bridging NFTs
+const myAccountKeyPair = KeyPair.fromString(myAccountPrivateKey);
+
+const res = await bridge.nftBridge(
+    ChainType.NEAR, // the side where we are locking token, in this example we are locking token on the Near side and bridging it to the Calimero side
+    myAccountId, // the account that is bridging NFT to the other side
+    myAccountKeyPair, // key pair of the account that is briding NFT
+    "paras-token-v2.testnet", // NFT contract id whose token we are bridging, note that the account that is bridging the asset needs to possess that asset (possess token with that token id) on that side.
+    "1976:57" // token id
+);
+console.log("NFT result: " + res);
+```
+
+### Withdraw NFTs to the original chain
+
+Now we will withdraw NFT that we bridged in the earlier example. For the withdrawal, we will use the ``` nftWithdraw ``` method.
+
+```js
+const accountId = "myaccount.testnet"; // same account as in the earlier example
+const calimeroPrivateKey = "ed25519:myaccount-private-key-on-calimero-side"; // note that this private key is different than the one in the example above since one Account does not share key pair across chains
+const keyPair = KeyPair.fromString(calimeroPrivateKey);
+
+await bridge.nftWithdraw(
+    ChainType.CALIMERO, // the side where we are burning and withdrawing NFTs, in this example, we are burning token on the Calimero side and unlocking it on the Near side (inverse action to the one in the previous example).
+    accountId, // the account that is withdrawing NFT to the other side
+    keyPair, // key pair of the account that is withdrawing NFT
```

Comment:
should we add in comment this is the keypair of Calimero account holding the nft?
e.g.
``
    keyPair, // key pair of the Calimero account that is withdrawing NFT
``
I mean it can also be NEAR account if withdrawing from NEAR